Formula/arangodb.rb

```diff
@@ -11,81 +12,98 @@ class Arangodb < Formula
     sha256 "60d59e09f4a440cffeb077f1de14784519223e6f25ccaaf6df692dc63fd6cac8" => :mavericks
   end
 
-  head do
-    url "https://github.com/arangodb/arangodb.git", :branch => "unstable"
-    depends_on "cmake" => :build
-  end
-
+  depends_on "cmake" => :build
   depends_on "go" => :build
   depends_on "openssl"
 
   needs :cxx11
 
+  # malloc_usable_size is not supported
+  patch :DATA
+
   fails_with :clang do
     build 600
     cause "Fails with compile errors"
   end
 
+  resource "arangodb2" do
+    url "https://www.arangodb.com/repositories/Source/ArangoDB-2.8.10.tar.gz"
+    sha256 "3f2adba7bb4fe5a93959c6929b984b053e66d3653144ed6bb1cdedd017c65050"
+  end
+
+  resource "upgrade" do
+    url "https://www.arangodb.com/repositories/Source/upgrade3-1.0.0.tar.gz"
+    sha256 "965f899685e420530bb3c68ada903c815ebd0aa55e477d6949abba9506574011"
+  end
+
   def install
     ENV.libcxx
+    ENV.cxx11
 
-    if build.head?
-      mkdir "arangodb-build" do
-        system "cmake", "..", "-DHOMEBREW=On", "-DUSE_OPTIMIZE_FOR_ARCHITECTURE=Off", "-DASM_OPTIMIZATIONS=Off", "-DETCDIR=#{prefix}/etc", "-DVARDIR=#{var}", *std_cmake_args
-        system "make", "install"
-      end
+    (libexec/"arangodb2/bin").install resource("upgrade")
+
+    resource("arangodb2").stage do
+      ENV.libcxx
```

Comment:
> Yes, it gets pushed down...

Cool 👍. I thought it would otherwise we'd have seen some fairly bizarre behaviour between main build & resource build interaction by now.